---------------------------------------------------------
#doctest 3-say_my_name.py
---------------------------------------------------------
#Check if last_name and first_name be a string:

>>> say_my_name = __import__('3-say_my_name').say_my_name

>>> say_my_name("didier","raoult")
My name is didier raoult

>>> say_my_name("didier","raoult")
My name is didier raoult

>>> say_my_name("didier",'d')
My name is didier d

>>> say_my_name("didier")
My name is didier 

>>> say_my_name("", "didier")
My name is  didier

===========================================================

# Error :

>>> say_my_name(didier,raoult)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'didier' is not defined

>>> say_my_name(5, "didier")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/workspaces/holbertonschool/holbertonschool-higher_level_programming/python-test_driven_development/3-say_my_name.py", line 8, in say_my_name
    raise TypeError("first_name must be a string")
TypeError: first_name must be a string

>>> say_my_name("yves" , "didier", "raoult")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: say_my_name() takes from 1 to 2 positional arguments but 3 were given

>>> say_my_name([1,2,3] , "didier")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/workspaces/holbertonschool/holbertonschool-higher_level_programming/python-test_driven_development/3-say_my_name.py", line 8, in say_my_name
    raise TypeError("first_name must be a string")
TypeError: first_name must be a string

>>> say_my_name()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: say_my_name() missing 1 required positional argument: 'first_name'
